packages/client/src/interceptors.ts

```diff
@@ -25,6 +25,16 @@ export interface WorkflowStartInput {
   readonly options: CompiledWorkflowOptions;
 }
 
+/** Input for WorkflowClientInterceptor.update */
+export interface WorkflowUpdateInput {
+  readonly updateName: string;
+  readonly args: unknown[];
+  readonly workflowExecution: WorkflowExecution;
+  readonly firstExecutionRunId?: string;
+  readonly headers: Headers;
+  readonly options?: WorkflowUpdateOptions;
```

Comment:
`options` should be mandatory, even if it ends up being an empty object by default. Makes it easier to implement interceptors.

Reply:
:+1: user-facing default can even be `= { }` I am guessing?